truenas/webui
# NAS-116440 / 22.12 / NAS-116440: Reducing usages of GlobalActions

Testing:
- Toolbar in Reports
- Save debug in Advanced Settings
- Save configuration dialog when updating a system
- Toolbar in alert category

You may notice styling issues on some controls. This is related to https://jira.ixsystems.com/browse/NAS-116437
